catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -122,11 +122,29 @@ public UpdateRequest process(UpdateRequest input)
         if ((isValidatorEnforced(validatorName) && validationErrorsExist && enforceErrors)
             || isValidatorEnforced(validatorName) && validationWarningsExist && enforceWarnings) {
           INGEST_LOGGER.debug(
-              "The metacard with id={} is being removed from the operation because it failed the enforced validator [{}].",
+              "The metacard with title={} and id={} is being removed from the operation because it failed the enforced validator [{}].",
+              metacard.getTitle(),
               metacard.getId(),
-              validatorName);
+              validatorName,
+              e);
           return null;
         } else {
+          if (validationErrorsExist) {
+            INGEST_LOGGER.debug(
+                "The metacard with title={} and id={} had an unenforced validation error [{}].",
```

Comment:
✏️ The title can have spaces in some cases, so it'd look better with quotes. i.e. `title="{}"`

Reply:
fixed